docs/inventory.md

```diff
@@ -210,22 +210,34 @@ That diagram gives the following excerpt from the inventory for the `bastion` an
       hosts:
         bastion:
           ansible_host: 192.168.10.5
+          ansible_connection: local # if your are not running crucible from the bastion then remove this line
```

Comment:
I don't think the comment is needed here. It's not going to stop the PR but ansible_connection is basic ansible inventory stuff. Ultimately it makes the inventory messy and cluttered.

Reply:
I think leaving comments in the inventory is fine, although repeating the comment may be unnecessary here.

As soon as we wrap Crucible with something to simplify the UX of the interface (say a bash script or a python script) to get rid of the ansible-playbook command requiring multiple params, Ansible then becomes an implementation detail, something that is run in the background to do the heavy lifting throughout the deployment process.

Making things easier for people who don't have a lot of experience with Ansible by adding a sentence or two to the docs doesn't cost much, and facilitates transforming the goal from  "I want an OpenShift cluster deployed using Ansible" to "I want an OpenShift cluster deployed", which is an ultimately better goal. At least in my view :P

What I'd suggest is to remove this comment from the snippet as it's repeated multiple times, and instead, leave a note above the snippet explaining when and what `ansible_connection: local` is used for.